### PR TITLE
Require a problem description in issues

### PR DESCRIPTION
**Pull request by jankatins**
Currently some issues are just code examples without a description **why** a change should be done. This leads to problems when (years later) the current behaviour is questioned and no one can remember why it was changed.

### CODE REVIEW

**Comment by sinhrks** [comment]
I think it may better to be added:

- checkbox whether the reporter uses the latest version
  - reason otherwise

needs discussion though

**Comment by janschulz** [comment]
In Addition to the reason why the thing is acrually a problem?

**Comment by jreback** [comment]
@jorisvandenbossche @TomAugspurger

**Comment by janschulz** [comment]
See also https://github.com/pydata/pandas/pull/14031 for basically the same thing for PRs. That one also has a small example why I think such a description is needed.

**Comment by TomAugspurger** [comment]
Seems fine to me.

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+#### Problem description
+
+[this should explain **why** the current behaviour is a problem and why the expected output is a better solution.]
```
Comment (jreback) [inline .github/ISSUE_TEMPLATE.md]: what do the ``[ ]`` do here?

Comment (janschulz) [inline .github/ISSUE_TEMPLATE.md]: My idea was to indicate that this text should be replaced.

**Comment by jreback** [comment]
ok, let's see how this works.

thanks @janschulz

**Pull request closed without merging**